pom.xml

```
@@ -104,6 +104,7 @@
     <maven-war-plugin.version>3.3.2</maven-war-plugin.version>
     <mockito.version>5.2.0</mockito.version>
     <spotbugs-maven-plugin.version>4.7.3.4</spotbugs-maven-plugin.version>
+    <spotless-maven-plugin.version>2.35.0</spotless-maven-plugin.version>
```

The spotless plugin 2.36.0 was released April 6, 2023.
```suggestion
    <spotless-maven-plugin.version>2.36.0</spotless-maven-plugin.version>
```